backend/infrahub/core/diff/model/path.py

```diff
@@ -20,7 +20,7 @@
     from neo4j.graph import Node as Neo4jNode
     from neo4j.graph import Path as Neo4jPath
     from neo4j.graph import Relationship as Neo4jRelationship
-    from pendulum import Interval
+    from whenever import TimeDelta
```

Comment:
It would be nice to get rid of this import as well, if we can map it to a TimeDelta from our own TimeStamp class.

Reply:
agreed but it sounds like a bigger effort